package.json

```diff
@@ -33,13 +33,13 @@
     ],
     "readmeFilename" : "README.md",
     "dependencies" : {
-        "objex" : "0.3.x",
+        "objex" : "^0.4.0",
```

Comment:
In one case it's unnecessary, npm provide 0.4.1 or higher automatically. In other case — update to 0.4.1 was main goal of this PR.